mteb/tasks/Classification/fra/MovieReviewSentimentClassification.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 1024
+
+
+class MovieReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="MovieReviewSentimentClassification",
+        dataset={
+            "path": "allocine",
+            "revision": "a4654f4896408912913a62ace89614879a549287",
+        },
+        description="The Allociné dataset is a French-language dataset for sentiment analysis that contains movie reviews produced by the online community of the Allociné.fr website.",
+        reference="https://github.com/TheophileBlard/french-sentiment-analysis-with-bert",
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["fra-Latn"],
+        main_score="accuracy",
+        date=("2006-01-01", "2020-01-01"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="MIT",
+        socioeconomic_status="mixed",
+        annotations_creators="derived",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation=None,
```

Comment:
Seems like the repo has a suggested citation for the repo. I would use that (in bibtex format)

Reply:
Sure. For the points, 2+4 (for new language task) and 2 for review? just to make sure.

Reply:
yep!

Reply:
Hey we already have French classification tasks, so it's not the first task

Reply:
I believe the only one is the translated MassiveIntent?